modules/video/include/opencv2/video/tracking.hpp

```diff
@@ -887,6 +888,89 @@ class CV_EXPORTS_W TrackerNano : public Tracker
     //bool update(InputArray image, CV_OUT Rect& boundingBox) CV_OVERRIDE;
 };
 
+enum TrackState { New = 0, Tracked, Lost};
+
+class CV_EXPORTS Detection
+{
+public:
+    int classId;
+    float confidence;
+    cv::Rect box;
+};
+
+class CV_EXPORTS Strack {
```

Comment:
Please use `CV_EXPORTS_W` and `CV_WRAP` to make it available from Python,  Java and other binded languages.